views/show.ejs

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/public/style.css">
+    <title><%= resi.basics.name%>'s Resume</title>
+</head>
+<body>
+    <div>
```

Comment:
better semantic tags for user readability and accessibility like `header`, `article` etc 
or more descriptive classes/identifiers if you need to use `div`